usim_pytest/test_usimpy/test_compatibility.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from usim import Flag, time
+from usim.py import Interrupt
+
+from .utility import via_usimpy
+
+
+class TestUsim2Simpy:
+    @via_usimpy
+    def test_flag(self, env):
+        """Can trigger and set flags"""
+        flag = Flag()
+        assert not flag
+
+        def trigger_flag(after):
+            yield env.timeout(after)
+            yield flag.set()
+        env.process(trigger_flag(5))
+        yield flag
+        assert env.now == 5
+
+    @via_usimpy
+    def test_coroutine(self, env):
+        """Can yield-as-await coroutines"""
+        assert env.now == 0
+
+        async def ping_pong(value):
+            return value
+        result = yield ping_pong(3)
+        assert result == 3
+        assert env.now == 0
+
+    @via_usimpy
+    def test_time(self, env):
+        """Can yield-as-await time expressions"""
+        assert env.now == 0
+        yield (time + 3)
+        assert env.now == 3
+        yield (time == 10)
+        assert env.now == 10
+        yield (time >= 20)
+        assert env.now == 20
+        yield (time >= 20)
+        assert env.now == 20
+        yield (time >= 10)
+        assert env.now == 20
```

Comment:
Does it make sense to also test for failures? What would SimPy give for `yield (time == 10)`?

Reply:
I've added some failure tests. ``yield (time == 10)`` will just wait forever at that point, though.